lib/config/configuration.go

```diff
@@ -2747,6 +2747,14 @@ func Configure(clf *CommandLineFlags, cfg *servicecfg.Config, legacyAppFlags boo
 		cfg.Proxy.QUICProxyPeering = true
 	}
 
+	if rawPeriod := os.Getenv("TELEPORT_UNSTABLE_AGENT_ROLLOUT_SYNC_PERIOD"); rawPeriod != "" {
+		period, err := time.ParseDuration(rawPeriod)
+		if err != nil {
+			return trace.Wrap(err, "invalid agent rollout period %q: %v", rawPeriod, err)
```

Comment:
nit: Don't echo que error on trace.Wrap, it's already captured by the first parameter.

```suggestion
			return trace.Wrap(err, "invalid agent rollout period: %q", rawPeriod)
```